controller/src/controllers/lumigo_controller.go

```diff
@@ -583,6 +583,16 @@ func (r *LumigoReconciler) injectLumigoIntoResources(ctx context.Context, lumigo
 	}
 
 	for _, replicaset := range replicasets.Items {
+		ownedByDeployment := false
+		for _, ownerRef := range replicaset.OwnerReferences {
+			if ownerRef.Kind == "Deployment" {
+				ownedByDeployment = true
+			}
+		}
+		if ownedByDeployment {
+			log.Info("Found replicaset owned by deployment, not instrumenting", "name", replicaset.Name)
```

Comment:
There's no reason to have this at Info level, it's going to be very spammy. Make it at Debug please.